eras/shelley/impl/src/Cardano/Ledger/Shelley/Transition.hs

```diff
@@ -120,7 +123,12 @@ class
     SimpleGetter (TransitionConfig era) (PParams era)
   tcInitialPParamsG =
     to $ \tc ->
-      translateEra' (tc ^. tcTranslationContextL) (tc ^. tcPreviousEraConfigL . tcInitialPParamsG)
+      translateEra'
+        (tc ^. tcTranslationContextL)
+        (tc ^. tcPreviousEraConfigL . tcInitialPParamsG)
+
+  tcNetworkIDG :: SimpleGetter (TransitionConfig era) Network
+  tcNetworkIDG = tcShelleyGenesisL . to sgNetworkId
```

Comment:
Why do you need this is as a type class method? It will always have the same implementation. 

Extract it outside of `TransitionEra` type class.